libs/ui-lib/lib/common/utils.ts

```diff
@@ -59,5 +59,9 @@ export const getMaxFileSizeMessage = `File size is too big. The file size must b
 )}.`;
 
 export const validateFileName = (fileName: string) => {
-  return new RegExp(FILENAME_REGEX).test(fileName || '');
+  return (
+    new RegExp(FILENAME_REGEX).test(fileName || '') ||
+    fileName.indexOf('.yaml.patch') > -1 ||
```

Comment:
I don't think this is correct, because `FILENAME_REGEX` is ensuring that the fileName does not contain a slash. If one of the ORs becomes true, it could be for a fileName containing a slash.

I think we should use a second regular expression that contains the same part about the slash, and additionally it validates the yaml.patch or yml.patch part.

Also, note that `FILENAME_REGEX` is already a regular expression, so doing `new RegExp(regexp)` is unnecessary and less eficient.

Reply:
I've marked as draft this PR because BE is not allowing us to put .yaml.patch or .yml.patch in the name. I'm waiting for BE response to see how we can align the regexp.